docs/basics/101-3-modify.rst

```diff
@@ -0,0 +1,93 @@
+.. _modify:
+
+Starting from scratch: Modifying content
+----------------------------------------
+
+So far, we've only added new content to the dataset. And we haven't done
+much to that content up this point, to be honest. Lets see what happens if
+we add content, and then modify it.
+
+For this, in the root of ``DataLad-101``, create a simple text file
+called ``notes.txt``. This will contain all of the notes that you take
+throughout the course.
+
+Pick an editor of your choice and write a short summary of how to create
+a DataLad dataset from scratch. Below is one way to do it, using the
+``echo`` command that comes with any Unix system, and a simple redirection
+with ``>``.
+
+.. runrecord:: _examples/DL-101-40
+   :language: console
+   :workdir: dl-101
+   :realcommand: cd DataLad-101 && echo "One can create a new dataset with 'datalad create [--description] PATH'. The dataset is created empty" > notes.txt
+
+   $ echo "One can create a new dataset with 'datalad create [--description] PATH'. The dataset is created empty" > notes.txt
```

Comment:
Maybe instead

```
cat << EOT > notes.txt
One can create a new dataset with 'datalad create [--description] PATH'.
The dataset is created empty
EOT
```

Better formatting, less quoting. Just as magic ;-)

Reply:
I tried this, I think it doesn't work as good as echo. Even if I omit the leading ``$``, copy-pasting from the code-block to the terminal seems to copy some suboptimal formatting, and the command does not succeed:

```
cat << EOT > notes.txt
One can create a new dataset with 'datalad create [--description] PATH'.
The dataset is created empty
EOT.
heredoc else> 
```

that might not be something we want people to end up in.

Reply:
sorry, my bad - nevermind.

Reply:
this is done.